### ISSUE TITLE
Better UI Output Fields

### ISSUE THREAD

**Issue by HarshKhandeparkar** [publiclab/image-sequencer#1471]
### Please describe the problem (or idea)
Currently, module output fields in the UI look the same as the input fields.
![Screenshot from 2020-01-12 23-09-35](https://user-images.githubusercontent.com/34770591/72222999-a0f20600-3590-11ea-898d-5005117fde52.png)

We could add better output fields with a `copy` button instead of a text field that can be edited.

### Please show us where to look

https://beta.sequencer.publiclab.org

### What's your PublicLab.org username?

> This can help us diagnose the issue:

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by ataata107** [comment]
Can I claim this?

**Comment by HarshKhandeparkar** [comment]
Yeah, I guess. Wow! Your contributions are awesome! Just don't claim too many issues at the same time. Do one or two at a time. Don't strain yourself.

**Comment by ataata107** [comment]
Sure😅

**Comment by jywarren** [comment]
Oh awesome!
